contracts/actions/curve/locking/CurveLockAmount.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import "../../ActionBase.sol";
+import "../helpers/CurveHelper.sol";
+import "../../../utils/TokenUtils.sol";
+
+contract CurveLockAmount is ActionBase, CurveHelper {
```

Comment:
Can we find out if we have crv locked, so we have only 1 actions that will itself see if we need to add additional or add for the first time.